ui/src/app/views/workspace-edit/component.ts

```diff
@@ -48,17 +50,28 @@ export class WorkspaceEditComponent implements OnInit {
         population: false,
         reviewRequested: false
       }};
+    if (this.route.routeConfig.data.title === 'Create Workspace') {
```

Comment:
This seems pretty brittle. Is `routeConfig` just a copy of data we give to the router earlier -- can we put an extra flag in there? Or could we give the router a different componenet for creation v. editing, where one is a very thin subclass of the other that just adds `this.adding = true`?